rho/scancommand.py

```diff
@@ -239,7 +239,8 @@ def run_ansible_with_vault(cmd_string, vault_pass, ssh_key_passphrase=None,
                 child.logfile = logfile
                 child.expect(pexpect.EOF)
 
-            child.wait()
+            if child.isalive():
```

Comment:
I encountered an exception `raise ExceptionPexpect ('Cannot wait for dead child process.')` on the wait call. The ansible ping had completed by the time we hit the wait. Feel free to open an issue on this if you think this isn't appropriate.